opentaskspal/src/main/java/org/dmfs/android/contentpal/operations/AssertRow.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2017 dmfs GmbH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.dmfs.android.contentpal.operations;
+
+import android.provider.BaseColumns;
+
+import org.dmfs.android.contentpal.RowData;
+import org.dmfs.android.contentpal.RowSnapshot;
+import org.dmfs.android.contentpal.Table;
+import org.dmfs.android.contentpal.rowdata.Composite;
+import org.dmfs.android.contentpal.rowdata.RelatedData;
+
+
+/**
+ * @author Gabor Keszthelyi
+ */
+public final class AssertRow<T> extends DelegatingOperation<T>
+{
+
+    public AssertRow(Table<T> table, RowSnapshot<T> rowSnapshot, RowData<T> rowData)
+    {
+        super(new Counted<T>(1,
+                new BulkAssert<T>(table,
```

Comment:
This should use `Assert` now.

Reply:
This was removed as well while you were reviewing.